Formula/geoserver.rb

```diff
@@ -11,7 +11,11 @@ def install
     (bin/"geoserver").write <<~EOS
       #!/bin/sh
       if [ -z "$1" ]; then
-        echo "Usage: $ geoserver path/to/data/dir"
+        echo "Usage: $ geoserver -start | -stop | path/to/data/dir"
+      elif [ $1 == "-start" ]; then
+        cd "#{libexec}/bin" && sudo ./startup.sh
```

Comment:
I doubt whether we want to use `sudo` in the wrapper script. cc @MikeMcQuaid

Reply:
We definitely do not want to use `sudo` in the wrapper script.